### PR TITLE
Local Codepen support update to ngrok@5 with authentication

### PR DESCRIPTION
**Pull request by Shubhabrata08**
## Previous Behavior
Local codepen testing was not configured with authentication support
## New Behavior
ngrok@3 bumped up to ngrok@5 with authentication token support
Requisites:
ngrok account: The dev must set up an ngrok account and activate his/her authToken. A file named **.codepen_auth** is to be created in the package folder and the dev can simply paste his/her authToken in it.
image

## Related Issue(s)

- Fixes #

### CODE REVIEW

File: scripts/executors/local-codepen.js [hunk 1]
```diff
@@ -23,8 +23,9 @@ if (fs.existsSync(configPath)) {
   try {
     console.log("Attempting to npm link globally installed ngrok so it can be require'd");
     // This will probably install ngrok globally if it's not already present
-    execSync('npm link ngrok@3', { cwd: gitRoot, stdio: 'inherit' });
+    execSync('npm link ngrok@5 --force', { cwd: gitRoot, stdio: 'inherit' });
```
Comment (AtishayMsft) [inline scripts/executors/local-codepen.js:26]: Thanks for the POC.
Lets move the changes to local-codepen.js file and .gitignore file to a separate PR and push the changes only in react-charting package.
I will take a look how to make use of the existing script and if there are any prerequisites that we need to setup to be able to use the common script with minimal changes.

Comment (Shubhabrata08) [reply scripts/executors/local-codepen.js:26]: Ok....
Moving script to charting package then

Comment (AtishayMsft) [reply scripts/executors/local-codepen.js:26]: dont move local-codepen.js script to charting package. Just move it to a different PR.

Comment (Shubhabrata08) [reply scripts/executors/local-codepen.js:26]: Done

**Comment by layershifter** [comment]
Closing for housekeeping 🏡 Feel free to re-open

**Pull request closed without merging**